hadoop-ozone/dev-support/checks/_mvn_unit_report.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REPORT_DIR=${REPORT_DIR:-$PWD}
+
+## generate summary txt file
+find "." -name 'TEST*.xml' -print0 \
+    | xargs -n1 -0 "grep" -l -E "<failure|<error" \
+    | awk -F/ '{sub("'"TEST-"'",""); sub(".xml",""); print $NF}' \
+    | tee "$REPORT_DIR/summary.txt"
+
+#Copy heap dump and dump leftovers
+find "." -name "*.hprof" -exec cp {} "$REPORT_DIR/" \;
+find "." -name "*.dump" -exec cp {} "$REPORT_DIR/" \;
```

Comment:
Please also copy `*.dumpstream`, which may help finding out the cause of `Corrupted STDOUT` warning of forked JVM.

```suggestion
find "." -name "*.dump" -exec cp {} "$REPORT_DIR/" \;
find "." -name "*.dumpstream" -exec cp {} "$REPORT_DIR/" \;
```

http://maven.apache.org/surefire/maven-surefire-plugin/faq.html#dumpfiles